### DISCUSSION TITLE
Option to disable interactions with Chatterino (Disable scrolling, changing channels, resizing text)

### DISCUSSION THREAD

**Discussion by Trizze** [Chatterino/chatterino2#5513] [Ideas]
Sometimes, when I play games (mostly older ones) my mouse cursor travels outside of the screen. I don't see it, but it's there somewhere. I end up scrolling in Chatterino whenever I use my scroll wheel in the game.

(The game I noticed it in recently was Black and White 2)

**Solution idea:**
Enable Chatterino Lock (Disables changing channels/scrolling/resizing text)
[Yes] [No]

If enabled, a small box appears in the top left of Chatterino, which you can click to either lock/unlock the program.

**Comment by fourtf** [comment]
Since this is an issue with the games I don't see a strong reason for us to add this. Chatterino just happens to be the application on the second screen here. You could argue that any application running on a second screen should have this disable feature, so web browsers, messengers, spotify, etc. I don't find that compelling.

**Reply by Trizze** [reply]
I agree it's not the biggest deal but, I think Chatterino is way more important for the streamer than any other app, so I don't think other apps need this feature. Having the chat scroll by accident makes Chatterino unusable and forces the user to use another chat program.

Maybe it's because I'm speaking from a streamer point of view though.
